test/boilerplate_suggestors/props_mixin_migrator_test.dart

```diff
@@ -12,23 +12,42 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-import 'dart:convert';
-
 import 'package:over_react_codemod/src/boilerplate_suggestors/boilerplate_utilities.dart';
 import 'package:over_react_codemod/src/boilerplate_suggestors/props_mixins_migrator.dart';
 import 'package:test/test.dart';
 
 import '../util.dart';
-import 'boilerplate_utilities_test.dart';
 
-main() {
+void main() {
   group('PropsMixinMigrator', () {
-    final converter = ClassToMixinConverter();
-    final testSuggestor = getSuggestorTester(PropsMixinMigrator(converter));
+    PropsMixinMigratorTestHelper();
+
+    group('with --treat-all-components-as-private flag', () {
+      PropsMixinMigratorTestHelper(
+        shouldTreatAllComponentsAsPrivate: true,
+      );
+    });
 
-    setUpAll(() {
-      semverHelper = SemverHelper(jsonDecode(reportJson));
+    group('with invalid file path', () {
+      PropsMixinMigratorTestHelper(
+        path: 'test/boilerplate_suggestors/does_not_exist.json',
+        isValidFilePath: false,
+      );
     });
+  });
+}
+
+void PropsMixinMigratorTestHelper({
```

Comment:
Only classes should be `UpperCamelCase`

```suggestion
void propsMixinMigratorTestHelper({
```